kolla-builder/README.md

```diff
@@ -0,0 +1,36 @@
+# kolla-builder
+[![Docker Repository on Quay](https://quay.io/repository/v1k0d3n/kolla-build/status "Docker Repository on Quay")](https://quay.io/repository/v1k0d3n/kolla-build)
```

Comment:
Needs to be changed to https://quay.io/charter-os/kolla-build
Latest version is v0.1.3